content/20.treatments.md

```diff
@@ -571,4 +571,37 @@ Summarize the state of the neutralizing antibody approach.
+For example, hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV [@doi:10.1021/jm0601856].
+Future work could explore whether these compounds exert antiviral effects against SARS-CoV-2 and if they are safe for use in animals and humans.
+
+The tocilizumab trial described in an above section [@doi:10/ggnxb3] studies the possibility of using an anti-inflammatory drug, typically used for the treatment of autoimmune disease, to counter the effects of the “cytokine storm” induced by the virus.
+Another anti-IL-6 antibody, sarilumab, is also being investigated [how to cite? http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19 or https://clinicaltrials.gov/ct2/show/NCT04327388].
+Typically, immunosuppressive drugs are often contraindicated in the setting of infection [@doi:10.1177/2040622313485275].
+However, when the need to curtail the “cytokine storm” inflammatory response to the virus transcends the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.
+COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, suggesting that immunosuppression may be a helpful approach to treatment [@doi:10.1016/S0140-6736(20)30628-0].
+While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-alpha. Inhibition of TNF-alpha by an inhibitor such as Etanercept has been previously suggested for treatment of SARS-CoV [@doi:10.1185/030079903125002757] and may be relevant for SARS-CoV-2 as well.
+Baricitinib and other small molecule inhibitors of the JAK kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections [@doi:10.1016/S1473-3099(20)30132-8].
+Baricitinib in particular may be able to reduce the ability of SARS-CoV-2 to infect lung cells [@doi:10.1016/S0140-6736(20)30304-4].
+Clinical trials studying baricitinib in COVID-19 have already begun in the US and in Italy (how to cite? US: https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19, Italy: https://clinicaltrials.gov/ct2/show/NCT04320277).
+Identification and targeting of further inflammatory markers that are relevant in SARS-CoV-2 infection may be of value for curtailing the inflammatory response and lung damage.
+
+Another approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
+The potential for remdesivir as an antiviral has already been described in an above section. Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses. Intriguing new options are emerging, however.
```

Comment:
```suggestion
The potential for remdesivir as an antiviral has already been described in an above section. 
Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses. 
Intriguing new options are emerging, however.
```

Reply:
Is it worth mentioning here that in severe cases of COVID-19 using antivirals such as remdesivir might be futile as at that points the main aim is to reduce the cytokine storm ensuing. 

just a thought.

Reply:
This makes sense to me as well, thanks for the suggestion. I just added a sentence at line 596, let me know what you think.